core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -171,6 +176,31 @@ describe('Library index page', function() {
     users.logout();
   });
 
+  it('should save exploration to play later list', function() {
```

Comment:
Do we have a file which contains "LearnerDashboard" released e2e tests? (I think that would be a better place for these tests.)

Reply:
Yeah, there's a learnerDashboard e2e test file. So shall I move the test there?

Reply:
Yep let's move it there*